src/Drupal/Commands/core/DeployHookCommands.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+namespace Drush\Drupal\Commands\core;
+
+use Consolidation\Log\ConsoleLogLevel;
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+use Consolidation\SiteAlias\SiteAliasManagerAwareTrait;
+use Consolidation\SiteAlias\SiteAliasManagerAwareInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\Core\KeyValueStore\KeyValueFactoryInterface;
+use Drupal\Core\Update\UpdateRegistry;
+use Drupal\Core\Utility\Error;
+use Drush\Commands\DrushCommands;
+use Drush\Exceptions\UserAbortException;
+use Psr\Log\LogLevel;
+
+class DeployHookCommands extends DrushCommands implements SiteAliasManagerAwareInterface
+{
+    use SiteAliasManagerAwareTrait;
+
+    /**
+     * @var \Drupal\Core\Update\UpdateRegistry
+     */
+    protected $registry;
+
+    /**
+     * @var \Drupal\Core\KeyValueStore\KeyValueStoreInterface
+     */
+    protected $keyValue;
+
+    /**
+     * DeployHookCommands constructor.
+     *
+     * @param string $root
+     *   The app root.
+     * @param string $site_path
+     *   The site path.
+     * @param \Drupal\Core\Extension\ModuleHandlerInterface $moduleHandler
+     *   The module handler.
+     * @param \Drupal\Core\KeyValueStore\KeyValueFactoryInterface $keyValueFactory
+     *   The key value store factory.
+     */
+    public function __construct($root, $site_path, ModuleHandlerInterface $moduleHandler, KeyValueFactoryInterface $keyValueFactory)
+    {
+        parent::__construct();
+        $this->keyValue = $keyValueFactory->get('deploy_hook');
+        $this->registry = new class(
+            $root,
+            $site_path,
+            array_keys($moduleHandler->getModuleList()),
+            $this->keyValue
+        ) extends UpdateRegistry {
+            public function setUpdateType($type)
+            {
+                $this->updateType = $type;
+            }
+        };
+        $this->registry->setUpdateType('deploy');
+    }
+
+    /**
+     * Prints information about pending deploy update hooks.
+     *
+     * @usage deploy:hook-info
```

Comment:
How about hook-status instead of hook-info? Thats consistent with updatedb:status and pm-updatestatus (drush8)

Reply:
makes sense